pkgs/development/python-modules/py3nvml/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, xmltodict
+}:
+
+buildPythonPackage rec {
+  pname = "py3nvml";
+  version = "0.2.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1jyplsmj85alwzypnd1iy40hg88p6zv6aw80lamq2sqkay5nqhcq";
+  };
+
+  buildInputs = [ xmltodict ];
+
```

Comment:
Switch to the GitHub release as there are [tests](https://github.com/fbcotter/py3nvml/tree/master/tests). You you can't run them add `  doCheck = false;`. Additionally add `pythonImportsCheck`.

Reply:
I've decided to take this out of this PR and I'll create another one for that package since this requires Nvidia `linuxPackages.nvidia_x11`.and some other stuff.